src/main/kotlin/no/nav/familie/klage/fagsak/dto/FagsakDto.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.klage.fagsak.dto
+
+import no.nav.familie.klage.fagsak.domain.Fagsak
+import no.nav.familie.klage.fagsak.domain.Stønadstype
+import java.util.UUID
+
+data class FagsakDto(
+    val id: UUID,
+    val fagsakPersonId: UUID,
```

Comment:
Trenger vi `FagsakPerson` i klage?
Vi forholder oss vel til en fagsakId her, og trenger egentlige ikke FagsakPerson?

Reply:
Vi bruker det vel for å evt. håndtere flere identer på tvers av ytelsene, men kunne sikkert vært løst uten overbygget. Jeg lar det stå for nå og hører med Charlie hva som var grunnen for at vi tok det med :)